reconcile/test/utils/test_slo_gatekeeper.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from pytest_mock import MockerFixture, MockFixture
+
+from reconcile.gql_definitions.fragments.saas_slo_document import (
+    AppV1,
+    ClusterV1,
+    NamespaceV1,
+    SaasSLODocument,
+    SLODocumentSLOSLOParametersV1,
+    SLODocumentSLOV1,
+    SLONamespacesV1,
+)
+from reconcile.gql_definitions.fragments.vault_secret import VaultSecret
+from reconcile.utils.secret_reader import SecretReaderBase
+from reconcile.utils.slo_gatekeeper import SLOGateKeeper
+
+
+@pytest.fixture
+def secret_reader(mocker: MockerFixture) -> MockerFixture:
```

Comment:
No need to create this fixture again, just use the default one in `tests/conftest.py`